static/app/components/tag.tsx

```diff
@@ -24,6 +24,10 @@ interface Props extends React.HTMLAttributes<HTMLSpanElement> {
    * Icon on the left side.
    */
   icon?: React.ReactNode;
+  /**
+   * Is tag used for a Promotion
+   */
+  isPromo?: boolean;
```

Comment:
@roggenkemper Can we make the props more generic and not mention promotions? Theoretically, you could use the new props for uses outside of promotions. Better to not have anything related to promotions in Sentry.

Reply:
It looks like there is already a tag `type="promotion"` - is this what you want?

Reply:
yes! ty for pointing that out